SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -1308,4 +1315,54 @@ extension PositioningViewController {
     }
 }
 
+extension PositioningViewController {
+    func prepareMarkerStairs() {
+        let icon = self.imageWithImage(
+            image: UIImage(named: "change_floor")!,
+            scaledToSize: CGSize(width: 40.0, height: 40.0)
+        )
+        let markerView = UIImageView(image: icon)
+        self.stairs.iconView = markerView
+        self.stairs.position = CLLocationCoordinate2D(latitude: 0, longitude: 0)
+        self.stairs.map = nil
 
+    }
+    
+    func showMarkerStairs(position: CLLocationCoordinate2D) {
+        self.stairs.position = position
+        self.stairs.map = self.mapView
+    }
+    
+    func imageWithImage(image:UIImage, scaledToSize newSize:CGSize) -> UIImage{
```

Comment:
imageWithImage non deixa claro que fai este método. A ver se podes buscar un nóme de método que se adecue máis a https://www.swift.org/documentation/api-design-guidelines/#conventions.